analyzer/server.go

```diff
@@ -207,6 +221,12 @@ func (s *Server) RegisterRpcEndpoints() {
 			"/rpc/flows",
 			s.FlowSearch,
 		},
+		{
+			"LayerConversation",
+			"GET",
+			"/data/conversation.json/{layer}",
```

Comment:
yeah gorilla supports it, take a look a the REAME. I'm ok it's not RPC but it's quite usual to put the extension at the end in order to say the output will be in such format or such format. Since we are going to probably return always json, I suggest to remove the extension!